test/integration/organizations/staff/adoption_application_reviews_test.rb

```diff
@@ -61,6 +79,40 @@ class Organizations::Staff::AdoptionApplicationReviewsTest < ActionDispatch::Int
       assert_equal("some notes", @under_review_app.notes)
     end
 
+    context "viewing application details" do
```

Comment:
Personally, I am no longer a fan of asserting div elements in tests because it makes the overhead of changing the UI much higher. Can we remove these ones in this integration test?

Reply:
Have removed assert select from this as well as controller test to remove dependency on UI

Reply:
Thank you!